system76/gaze18/default.nix

```diff
@@ -0,0 +1,37 @@
+{ config, lib, pkgs, ... }:
+{
+  imports = [
+    ../.
+  ];
+  boot.initrd.kernelModules = [ "nvidia" ];
+
+  hardware.opengl = {
+    enable = lib.mkDefault true;
+    driSupport = lib.mkDefault true;
+    driSupport32Bit = lib.mkDefault true;
+    extraPackages = with pkgs; [
+      vaapiVdpau
+    ];
+  };
+
+  services.xserver.videoDrivers = ["nvidia"];
+
+  hardware.nvidia = {
+
+    modesetting.enable = lib.mkDefault true;
+
+    powerManagement.finegrained = lib.mkDefault true;
```

Comment:
Some laptops have issues with suspend and this. I assume this is not the case here?

Reply:
Nope, I haven't encountered any issues with suspend.